README.md

```diff
@@ -23,12 +23,19 @@ conda activate cif-benchmark
 tar xf structures_0100.tar.gz
 ```
 
+## ToDo
+- [ ] test [computational crystallography toolbox](https://cctbx.github.io) 
+
 ## Benchmark
 
 ```
 ./benchmark.sh  # run all benchmarks
 snakeviz pycodcif_100.prof  # inspect one output
 ```
+or inspect on the command line 
```

Comment:
let's put this in the code section in the same style (command, followed by comment)